setup.py

```diff
@@ -26,13 +26,6 @@
 
 from setuptools import setup, find_namespace_packages
 
```

Comment:
Unused imports that can be removed here too:

- `from distutils.errors import DistutilsExecError`
- `from shutil import move, rmtree`

Reply:
Done.